gateway/src/apicast/policy/caching/schema.json

```diff
@@ -5,7 +5,7 @@
   "properties": {
     "exit": {
       "type": "caching_type",
-      "enum": ["resilient", "strict", "none"]
+      "enum": ["resilient", "strict", "allow", "none"]
```

Comment:
No need to change anything, but I wonder if there is a way to describe each individual enum value so we can show it nicely in the form.